documentation/README.md

```diff
@@ -84,39 +88,84 @@ To maximize screen real estate, Music Blocks overlays the program
+
+-   **Run Slowly**: Executes the program slowly to allow you to follow the process step by step.
+-   **Run Step by Step**: Runs the program one step at a time, ideal for debugging and analysis.
+-   **Display Statistics**: Displays project statistics such as performance metrics.
+-   **Beginner/Advanced Mode**: Switches between a simplified view for beginners and a more advanced view for experienced users.
+-   **Language Selector**: Allows you to change the language of the interface.
+
+---
+
+### 3. Palette Toolbar
+
+The **Palette** toolbar is located on the left side of the screen. It contains various categories of blocks, such as numbers, media, and actions, which you can drag and drop onto the canvas to build your project.
+
+The palette is dynamic, switching between different categories based on your selection. When you hover over the palette, it reveals the specific blocks related to your chosen category.
+
+---
+
+### 4. Canvas Toolbar (Upper Right)
+
+On the upper right of the canvas is a small toolbar with options for managing the workspace. It allows you to adjust the display and clear the canvas.
 
-(2) On the *Secondary* toolbar you will find the buttons *Run slowly*,
-*Run step by step*, *Display Statistics*, *beginner/advanced mode*,
-etc. and also the button for selecting language.
+-   **Show Grids**: Displays a grid overlay on the canvas for alignment and organization.
+-   **Clear Screen**: Clears the entire canvas, removing all blocks and drawings.
+-   **Toggle Display Size**: Adjusts the size of the canvas or blocks for a better viewing experience.
 
-(3) The *Palette* toolbar is on the left side of the screen. New
-blocks are dragged from the palette.
+---
 
-(4) On the upper right of the canvas is a small toolbar for showing
-grids, clearing the screen, and toggling the display size.
+### 5. Canvas Toolbar (Lower Right)
 
-(5) On the lower right of the canvas is a small toolbar where you will
-find the *Home* button, buttons for show/hide blocks, expand/collapse
-blocks and decrease/increase block size.
+Located at the lower right of the canvas, this small toolbar provides additional control over block visibility and size.
 
-These toolbars are described in detail in the
-[Turtle Blocks documentation pages](https://github.com/sugarlabs/turtleblocksjs/tree/master/documentation).
+-   **Home Button**: Takes you back to the initial view or project start screen.
+-   **Show/Hide Blocks**: Toggles the visibility of the blocks you’ve added to the canvas.
+-   **Expand/Collapse Blocks**: Allows you to expand or collapse the view of blocks for better organization.
+-   **Decrease/Increase Block Size**: Changes the size of the blocks on the canvas to suit your preference.
 
-An additional "contextual" menu appears whenever you *right click* on
-a block.
+---
+
+### 6. Contextual Menu
```

Comment:
There are two different contextual menus: one for when you click on a block (the one you describe) and one for when you click on the background, which includes most of the buttons found on the toolbars.

Reply:
@walterbender sir i have made the required changes please review it